portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java

```diff
@@ -34,18 +34,52 @@
 import com.google.appengine.api.datastore.Key;
 import com.google.appengine.api.datastore.KeyFactory;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+
 import com.google.sps.data.Comment;
+import com.google.sps.data.UserInfo;
 
 /** Servlet responsible for deleting comments. */
 @WebServlet("/delete-comment")
 public class DeleteCommentServlet extends HttpServlet {
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    long id = Long.parseLong(request.getParameter("id"));
+    response.setContentType("application/json");
 
-    Key taskEntityKey = KeyFactory.createKey("Comment", id);
+    long id = Long.parseLong(request.getParameter("id"));
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    datastore.delete(taskEntityKey);
+    Key taskEntityKey = KeyFactory.createKey("Comment", id);
+
+    //Only delete a comment if the author is trying to delete it. 
```

Comment:
nit: Space

Reply:
Assuming this is in regards to the vertical whitespace @ 56? If so, have removed. If not, I misunderstood.

Reply:
Actually I meant to add a space between // and Only.  Annoying that Github won't let me select individual parts of a line.   Comments are // comment.  Also a problem elsewhere in this PR.

Reply:
Oops! Fixing.

Reply:
Done.